### PR TITLE
allow additional ses entries

### PR DESCRIPTION
**Pull request by Jnig**

### CODE REVIEW

**Review by MattiasGees — CHANGES_REQUESTED** [review]
Can you also add the extra variable to the `README.md`?

**Comment by Jnig** [comment]
@MattiasGees sure, done :)

**Review by duboisph — APPROVED** [review]
Just a typo to fix: `Additional`

**Comment by Jnig** [comment]
ah thanks, done :)

File: r53.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ resource "aws_route53_record" "domain_amazonses_verification_record" {
   name    = "_amazonses.${var.domain}"
   type    = "TXT"
   ttl     = "3600"
-  records = ["${aws_ses_domain_identity.domain.verification_token}"]
+  records = ["${aws_ses_domain_identity.domain.verification_token}", "${var.ses_records}"]
```
Comment (duboisph) [inline r53.tf:7]: Actually, looking at it again, this probably won't work, have you tested it? You might need to use some interpolation function like `concat()`.

**Review by duboisph — CHANGES_REQUESTED** [review]

File: r53.tf (hunk 1 above)
Comment (Jnig) [reply r53.tf:7]: yes, i've tested it... ;)

Comment (Jnig) [reply r53.tf:7]: image

so you don't have to test it :)

Comment (Jnig) [reply r53.tf:7]: with ses_records = ["dsd", "dsda"]

**Review by MattiasGees — APPROVED** [review]

**Review by duboisph — APPROVED** [review]

**Pull request merged**